src/css/cs/items/_aa_planner.scss

```diff
@@ -5,9 +5,32 @@
       flex-wrap: wrap;
 
       > div {
-        background: $uc-planner-background;
+        background: $uc-color-white;
         font-size: 20px;
         margin: 10px;
+        max-width: 30%;
+      }
+    }
+  }
+}
+
+@media all and (max-width: 700px) {
+  .uc-planner {
```

Comment:
@marcpechaitis can we combine these statements?

Something along the lines of:

```
  .uc-planner {
    .uc-planner-detail {
      .uc-planner-detail-term {
        flex-direction: column;
        flex-wrap: nowrap;
        > div {
          max-width: 100%;
        }
      }
    }
  }
```

Also make sure to ping me with @christianv when you make an update to the PR.